sample/conn/grpc/server/server.go

```diff
@@ -99,7 +117,7 @@ func (s *server) Chat(stream proto.Channel_ChatServer) error {
 			if err == io.EOF {
 				break
 			} else if err != nil {
-				err = fmt.Errorf("error receiving from replica server stream: %s", err)
+				err = fmt.Errorf("Error receiving from server stream: %s", err)
```

Comment:
Thanks for pointing that out, I didn't know that convention. I think it would be better to keep a consistent style. Maybe we can change the style to recommended as a separate low-priority issue.

Reply:
What we could also do is to use "github.com/pkg/errors" package for wrapping error messages in the context of the call stack.